src/cmd-cleanup-bucket

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python3 -u
+
+'''
+    This script removes unreferenced builds from s3 bucket
```

Comment:
Hmm, I'd prefer something a bit more generic here, e.g. `cosa prune-cloud s3 ...`, like `buildprep` and `buildupload` do. That way, (1) we could even grow e.g. deleting AMIs for the S3 path, and (2) we leave the door open for other cloud storage/compute services.

Reply:
The only external place for the builds is s3 bucket, so it is being used as a source of truth. When build is removed from s3 bucket the script would attempt to remove all images (eventually) from all cloud services as long as references to AMIs / GCP images are stored in metadata

Reply:
Ahh OK, I missed that this was also nuking AMIs right now.

> The only external place for the builds is s3 bucket, so it is being used as a source of truth.

Right, but we can still trivially leave the door open for other clouds, right? E.g. `cosa remote-prune s3 ...` is just as fine, right? The `s3` there refers to where the source of truth is, not that we only prune things from S3.

Reply:
Ah, sure, that seems fair

Reply:
Implemented in 4aa52f7afb3655f80ec0a3d7e7d256623bddc7e2